changelog/@unreleased/pr-4188.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Allow disabling read-write conflict checking on a per transaction basis.
```

Comment:
Probably worth adding some warnings here, along the lines of `This feature may be dangerous and could result in SEVERE DATA CORRUPTION; if you don't know what you are doing please talk to the AtlasDB team first.`

Reply:
I will add that, but just to be clear, most people won't even have access to this. You need to be using the raw Transaction interface, most people use generated code and tables and such.

Reply:
Done